tests/equilibria/test_flux_surfaces.py

```diff
@@ -40,6 +41,44 @@
 TEST_PATH = get_bluemira_path("equilibria/test_data", subfolder="tests")
 
 
+class TestFluxSurface:
+    def setup_class(self):
+        eq_name = "DN-DEMO_eqref.json"
+        filename = os.sep.join([TEST_PATH, eq_name])
+        self.eq = Equilibrium.from_eqdsk(filename)
```

Comment:
If you want this read in once for all methods in this test class it should be the below alos use os.path over os.sep : 
```suggestion
    @classmethod
    def setup_class(cls):
        eq_name = "DN-DEMO_eqref.json"
        filename = os.path.join([TEST_PATH, eq_name])
        cls.eq = Equilibrium.from_eqdsk(filename)
```